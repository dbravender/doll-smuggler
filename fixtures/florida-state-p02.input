max weight: 26

available dolls:

name  weight  value
a         12     24
b          7     13
c         11     23
d          8     15
e          9     16
