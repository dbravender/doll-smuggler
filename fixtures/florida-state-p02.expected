packed dolls:

name   weight  value
d           8     15
c          11     23
b           7     13
